### PR TITLE
Command to delete flink jobs

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Don't let the papi db get cluttered without a way to clean it up. 

Fixes https://github.com/meroxa/turbine-project/issues/291

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

### CODE REVIEW

**Comment by justmisosoup** [comment]
@janelletavares How difficult possible would it be to rework the `meroxa flink` command to `meroxa jobs`?

**Comment by ericcheatham** [comment]
@justmisosoup It's a fairly low effort change overall. Though that does raise a question about how we should refer to jobs in documentation/help text. Instead of 
> Remove flink job

should it be something like

> Remove job

that feels non-specific.

cc @janelletavares

**Comment by justmisosoup** [comment]
@ericcheatham Good point. Articulating `Remove Flink job.` in the CLI documentation would be consistent with what we do with the `apps` commands. 

Today, that looks like:

```
Available Commands:
  deploy      Deploy a Turbine Data Application
  describe    Describe a Turbine Data Application
  init        Initialize a Turbine Data Application
  list        List Turbine Data Applications
  logs        View relevant logs to the state of the given Turbine Data Application
  ```

Looking forward to the future for when Flink is eventually encapsulated under `apps` we'll likely need to scrub Turbine Data Application from the CLI documentation and come up with something more generalised.

cc: @janelletavares

**Comment by janelletavares** [comment]
@justmisosoup I believe it's a one-liner to change `meroxa flink` to `meroxa jobs`. There are no generated docs for these commands yet because they are hidden, so nothing to update around there.

File: cmd/meroxa/root/flink/remove_test.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+			t.Fatalf("expected \"%s\" got \"%s\"", tt.err, err)
+		}
+
+		if tt.name != cc.args.NameOrUUID {
+			t.Fatalf("expected \"%s\" got \"%s\"", tt.name, cc.args.NameOrUUID)
+		}
+	}
+}
+
+func TestRemoveFlinkJobExecution(t *testing.T) {
```
Comment (jayjayjpg) [inline cmd/meroxa/root/flink/remove_test.go:59]: Nice test coverage! :sparkles:

File: utils/tests.go [hunk 2]
```diff
@@ -74,6 +74,13 @@ func GenerateResource() meroxa.Resource {
 	return GenerateResourceWithNameAndStatus("", "")
 }
 
+func GenerateFlinkJob() meroxa.FlinkJob {
```
Comment (jayjayjpg) [inline utils/tests.go:77]: I see, I was already wondering how you can mock out a flink job in those unit tests :eyes:

**Review by jayjayjpg — APPROVED** [review]
This reads great, thank you for putting this work together so quickly!

**Pull request merged**